tests/api/fixtures/users.js

```diff
@@ -88,6 +89,15 @@ const API = module.exports = {
   getTwoFriends: () => {
     twoFriendsPromise = twoFriendsPromise || getTwoFriends()
     return twoFriendsPromise
+  },
+
+  getRandomPosition: () => {
```

Comment:
maybe some DRYing with https://github.com/inventaire/inventaire/blob/master/tests/api/utils/users.js#L6

Reply:
done in geo-queries